sources/api/schnauzer/src/v2/import/helpers.rs

```diff
@@ -66,6 +66,7 @@ fn all_helpers() -> HashMap<ExtensionName, HashMap<HelperName, Box<dyn HelperDef
             "default" => helper!(handlebars_helpers::default),
             "join_array" => helper!(handlebars_helpers::join_array),
             "join_map" => helper!(handlebars_helpers::join_map),
+            "if-not-null" => Box::new(handlebars_helpers::IfNotNullHelper),
```

Comment:
It looks like the naming convention is underscore:

```suggestion
            "if_not_null" => Box::new(handlebars_helpers::IfNotNullHelper),
```

Reply:
the naming convention is a bit all over the place (see `ecr-prefix`, `tuf-prefix` above), but I'm good to stick with the convention within `std`

Reply:
There used to be a comment to help us remember which way we are supposed to name them. I know because I wrote it because Tom told me to! Maybe this comment should be restored!

https://github.com/bottlerocket-os/bottlerocket/blob/b530f30850d58ad84a8b0ded2e08fa91d554c7a8/sources/api/schnauzer/src/lib.rs#L121

Reply:
I need to fix the commit title to be snake case too so I'll add this back